sas9.4-viya/azuredeploy.json

```diff
@@ -0,0 +1,3683 @@
+                    }
+                }
+            }
+        },
+        {
+            "type": "Microsoft.Resources/deployments",
+            "apiVersion": "2019-10-01",
+            "name": "Phase2-CASControllerViyaARK",
+            "dependsOn": [
+                "[resourceId('Microsoft.Resources/deployments', 'Phase1-CASControllerHostSetup')]",
+                "[resourceId('Microsoft.Resources/deployments', 'Phase1-AnsibleHostSetup')]"
+            ],
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[variables('linux_extension_template')]"
+                },
+                "parameters": {
+                    "location": {
+                        "value": "[parameters('location')]"
+                    },
+                    "vmName": {
+                        "value": "[variables('Cascontroller_vm_name')]"
+                    },
+                    "commandToExecute": {
+                        "value": "[concat('set -o pipefail;','./viya_ark.sh 2>&1 | tee /var/log/sas/install/runPhase2_viyaark.log')]"
+                    },
+                    "_artifactsLocation": {
+                        "value": "[variables('viya_ark_sh')]"
+                    },
+                    "_artifactsLocationSasToken": {
+                        "value": "[variables('_artifactsLocationSasToken')]"
+                    }
+                }
+            }
+        },
+        {
+            "type": "Microsoft.Network/networkInterfaces",
+            "apiVersion": "2019-09-01",
+            "name": "[concat(parameters('SAS Application Name'),'_',variables('casworker_vm_name'),copyIndex(),'_nic_',variables('resourceGroupUniqueString'))]",
+            "location": "[parameters('location')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Network/networkSecurityGroups', variables('casworker_nw_sg'))]",
+                "[resourceId('Microsoft.Network/virtualNetworks',variables('vnet_name'))]"
+            ],
+            "properties": {
+                "ipConfigurations": [
+                    {
+                        "name": "ipconfig1",
+                        "properties": {
+                            "privateIPAllocationMethod": "Dynamic",
+                            "subnet": {
+                                "id": "[resourceId('Microsoft.Network/virtualNetworks/subnets', variables('vnet_name'), variables('vnet_viya_pvt_subnt'))]"
+                            },
+                            "privateIPAddressVersion": "IPv4"
+                        }
+                    }
+                ],
+                "enableAcceleratedNetworking": true,
+                "enableIPForwarding": false,
+                "networkSecurityGroup": {
+                    "id": "[resourceId('Microsoft.Network/networkSecurityGroups', variables('casworker_nw_sg'))]"
+                }
+            },
+            "copy": {

```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties - fix all instances of this

Reply:
Sorted the code in order according to the best practices guide